shared/js/background/tab-manager.es6.js

```diff
@@ -5,6 +5,7 @@ const browserWrapper = require('./wrapper.es6')
 
 class TabManager {
     constructor () {
+        /** @type {{ [key: number]: Tab }} */
```

Comment:
NAB: prefer `Record<number, Tab>` here to indicate this type shouldn't inherit all of Object

Reply:
Thanks, agreed that's even better. I avoided changing this to a Map which would provide similar benefits to this as I understand it.